overrides/windows-override.json

```diff
@@ -68,7 +68,12 @@
             "state": "enabled"
         },
         "duckPlayer": {
-            "state": "enabled"
+            "state": "enabled",
+            "features": {
+                "pip": {
+                    "state": "disabled"
+                }
+            }
```

Comment:
I added it so we would be able to turn off the feature for Windows only if needed without affecting MacOs or other platforms. Vice-versa applies as well, it can turned on for Windows but not for MacOS.

Reply:
If that's the desired outcome, then adding to `windows-override.json` seems to be the better alternative:
- we only touch the windows override which I can approve compared to this PR where we'll need someone else too as it affects the main feature folder
- macOS can decide on whether on the feature themselves by adding it to their override (undefined feature will be treated as disabled)